### DISCUSSION TITLE
blog/learn-something/anki-is-all-you-need

### DISCUSSION THREAD

**Discussion by giscus** [roitium/NaiveBlog#57] [Announcements]
# blog/learn-something/anki-is-all-you-need

Anki，爽！

https://www.yanyaolab.xyz/blog/learn-something/anki-is-all-you-need

**Comment by L-M-Sherlock** [comment]
我是不是在别的地方见过你？

**Reply by roitium** [reply]
是这样的 hhh，之前写过一篇背单词的文章，不过最近一段重构了博客，评论没迁移。

~~另外是怎么搜到这篇文章的，我试了各种关键词都没搜到🤣~~

**Reply by L-M-Sherlock** [reply]
fsrs anki, 最近一周
